include/gps.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef OBC_GPS_HPP
+#define OBC_GPS_HPP
+
+#include <Adafruit_GPS.h>
+
+#include <array>
+
+#include "error.hpp"
+
+namespace obc {
+
+constexpr bool GPSECHO = false;
```

Comment:
złe konwencja dla nazwy stałej. powinno byc gpsecho jesli to jest jakis skrót albo gps_echo